src/Perlang.Parser/Scanner.cs

```diff
@@ -63,6 +63,7 @@ public class Scanner
                 { "byte", RESERVED_WORD },
                 { "sbyte", RESERVED_WORD },
                 { "short", RESERVED_WORD },
+                { "bigint", RESERVED_WORD },
```

Comment:
This is a drive-by fix. We should move this part to a separate PR and add some tests, just like we have for other not-yet-implemented keywords.

Reply:
Fixed in #238, resolving.